_episodes/025-katharina-florian.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "RustFest 2019 Interview Series: Burnout in Open Source Software; The Rust Roadmap"
+date: 2015-07-30T22:00:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e025-rustfest-katharina-florian.mp3
+duration: "49:03"
+length: "35318701"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+Two more long-awaited interviews from RustFest 2019: [Katharina Fey](https://twitter.com/spacekookie) on the phenomenon of burnout in software and in open source communities and [Florian Gilcher](https://twitter.com/Argorak) on Rust's annual roadmaps.
+
+<!--
+The episode introduction goes here.
+The first paragraph should ideally be short, and is used in various
+places as a "short description" for the episode. Any subsequent
+paragraphs show up as "expanded description".
+-->
+
+### Contributing to Rustacean Station
+
+<!-- You can probably leave this as-is -->
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+#### [@00:50] Part 1: Burnout w/ Katharina Fey
+
+ - [@01:54] - How common is burnout in software?
+ - [@03:24] - How does burnout manifest in volunteer endeavors like open source software?
+ - [@08:10] - How does rotation of responsibilities alleviate burnout?
+ - [@13:41] - What communities succeed at combating burnout?
+ - [@16:44] - Final thoughts on burnout and governance
+
+#### [@19:50] Part 2: The Rust Roadmap w/ Florian Gilcher
+
+<!--
```

Comment:
I wonder — are there useful links in what you talked to Florian about? Like a link to the roadmap possibly?

Reply:
I could link to the 2019 roadmap, I suppose I could link to the 2020 roadmap as well although at the time it didn't exist, so we were mostly just talking about it in abstract, would that be weird?

Reply:
I think linking to both would be a good idea! It's something the listeners are likely to care about after listening to it :)

Reply:
Gentle nudge on this